rds.tf

```diff
@@ -1,10 +1,11 @@
 resource "aws_rds_cluster" "awsRdsNotEncrypted" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Use Encrypted RDS storage**
 
**Vulnerability**
Encrypt Amazon RDS instances and snapshots at rest, by enabling the encryption option for your Amazon RDS DB instance
 
**Remediation**
On the AWS RDS console, for each RDS instance that failed the rule, enable encryption in the Details section. You can choose to use your default AWS encryption key, or supply an AWS KMS key. Refer to: http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.Encryption.html